develop/entities/effects.md

```diff
@@ -57,10 +58,30 @@ language file.
 }
 ```
 
-### Testing {#testing}
+### Applying The Effect {#applying-the-effect}
 
-Use the command `/effect give @p fabric-docs-reference:tater` to give the player our Tater effect.
-Use `/effect clear @p fabric-docs-reference:tater` to remove the effect.
+It's worth taking a look at how you'd typically apply an effect to an entity.
+
+::: tip
+For a quick test, it might be a better idea to use the previously mentioned `/effect` command: <br/>
+`/effect give @p fabric-docs-reference:tater`
+:::
+
+To apply an effect internally, you'd want to use the `LivingEntity#addStatusEffect` method, which takes in
+a `StatusEffectInstance`, and returns a boolean, specifying whether the effect was successfully applied.
+
+```java
+var instance = new StatusEffectInstance(FabricDocsReferenceEffects.TATER, 5 * 20, 0, false, true, true);
```

Comment:
This should really be in the reference mod, even just in the entrypoint if nessecary

Reply:
The thing is, where should I put it?

Reply:
the only logical thing would be a static grantEffect() method, or something similar